src/gradient/gradient.class.ts

```diff
@@ -131,7 +133,7 @@ export class Gradient<S, T extends GradientType = S extends GradientType ? S : '
       gradientUnits: this.gradientUnits,
       gradientTransform: this.gradientTransform ? this.gradientTransform.concat() : this.gradientTransform
     };
-    populateWithProperties(this, object, propertiesToInclude);
+    fabric.util.populateWithProperties(this, object, propertiesToInclude);
```

Comment:
why revert?

Reply:
Will be handled by #8202

Reply:
reverted because i didn't conver it yet so is not importable. That was the reason. The original index.ts is removed for now because it causes me issues while converting